lib/activity/router.js

```diff
@@ -76,6 +77,13 @@ module.exports = function route(callback) {
         // Delay GitHub API calls to avoid replication lag
         context.github.hook.before('request', avoidReplicationLag());
 
+        const githubName = subscription.type === 'account'
+          ? context.payload.repository.owner.login
+          : context.payload.repository.full_name;
+        if (subscription.githubName !== githubName) {
+          await subscription.update({ githubName });
+        }
+
```

Comment:
I wonder if it makes sense to have this logic in the `Subscription` model as a method like `updateName`?

Reply:
Sounds good. Done!